packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -32,30 +32,6 @@ export interface ModelInstanceDocumentMetadataArgs {
    */
   controllers?: Array<string>
 
```

Comment:
```suggestion
  controllers?: Array<string>

  model: StreamID
```

A MID always need a model.

Reply:
I think the goal of this PR was just to remove no-longer-needed functionality.  Adding in the model to the APIs will happen in https://linear.app/3boxlabs/issue/NET-1446/bring-modelinstancedocument-public-api-into-alignment-with-the-spec